From 3834b8d7d9c25f7efc1d8c301b5452a659d27f10 Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Sun, 28 May 2023 22:07:45 +0300
Subject: [PATCH 101/101] dtb: bluetooth node deletion fix

Delete bt@uart4 node to prevent appearance of an inoperable bt device.

Reported-by: Denis Bublik <denis.bublik@compulab.co.il>
Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts | 1 -
 arch/arm64/boot/dts/compulab/sb-iotgimx8.dts     | 4 ++++
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts b/arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts
index 22f93994ac05..26d983fa925a 100644
--- a/arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts
+++ b/arch/arm64/boot/dts/compulab/sb-iotgimx8-ied.dts
@@ -106,7 +106,6 @@
 	pinctrl-0 = <&pinctrl_uart4 &pinctrl_uart4_gpio>;
 	/delete-property/ uart-has-rtscts;
 	status = "disabled";
-	/delete-node/ bluetooth;
 };
 
 &ecspi2 {
diff --git a/arch/arm64/boot/dts/compulab/sb-iotgimx8.dts b/arch/arm64/boot/dts/compulab/sb-iotgimx8.dts
index 3e428fab3aa3..7e917c6669d8 100644
--- a/arch/arm64/boot/dts/compulab/sb-iotgimx8.dts
+++ b/arch/arm64/boot/dts/compulab/sb-iotgimx8.dts
@@ -163,6 +163,10 @@
 	status = "okay";
 };
 
+&uart4 {
+	/delete-node/ bluetooth;
+};
+
 &usbotg1 {
 	dr_mode = "host";
 	disable-over-current;
-- 
2.17.1

